core/src/main/java/org/frankframework/http/authentication/AbstractClientCredentials.java

```diff
@@ -0,0 +1,68 @@
+/*
+   Copyright 2024 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package org.frankframework.http.authentication;
+
+import lombok.extern.log4j.Log4j2;
+
+import org.apache.http.NameValuePair;
+import org.apache.http.auth.Credentials;
+import org.apache.http.client.methods.HttpEntityEnclosingRequestBase;
+import org.apache.http.message.BasicNameValuePair;
+
+import org.frankframework.configuration.ConfigurationException;
+import org.frankframework.configuration.ConfigurationWarnings;
+import org.frankframework.http.AbstractHttpSession;
+
+import java.util.List;
+
+@Log4j2
+public abstract class AbstractClientCredentials extends AbstractOauthAuthenticator {
```

Comment:
Deze hoeft toch niet abstract te zijn

Reply:
Dat is de class die geinherit wordt door `ClientCredentialsBasicAuth` en `ClientCredentialsQueryParameters`. Dat zijn de echte implementaties.